src/frontend/src/components/ReactHookEditableList.tsx

```diff
@@ -1,70 +1,64 @@
+  name,
+  ls,
+  register,
+  append,
+  remove,
+  bulletName = 'Item'
+}) => {
   return (
-    <>
-      <Stack spacing={1}>
-        <Stack direction={'row'} spacing={2}>
-          <Typography variant="h5">{title}</Typography>
-          <Button
-            variant="contained"
-            color="primary"
-            onClick={() => append({ bulletId: -1, detail: '' })}
-            sx={{ my: 2, width: 'max-content', textTransform: 'none' }}
-          >
-            + Add {title}
-          </Button>
-        </Stack>
-        <Grid container spacing={2}>
-          {
-            //</Grid><Grid container sx={{ display: 'flex', flexDirection: 'row', flexWrap: 'wrap' }} spacing={2}>
-          }
-          {ls.map((_element, i) => {
-            return (
-              <>
-                <Grid item sx={{ alignItems: 'center' }} xs={12} md={4}>
-                  <InputLabel>{title + ' ' + (i + 1)}</InputLabel>
-                  <Box sx={{ display: 'flex' }}>
-                    <TextField
-                      required
-                      multiline
-                      placeholder={'Enter ' + title + '...'}
-                      inputProps={{ style: { fontSize: 14 } }}
-                      minRows={2}
-                      autoComplete="off"
-                      sx={{ width: 12 / 12, padding: 0 }}
-                      {...register(`${name}.${i}.detail`)}
-                    />
-                    <Stack spacing={0}>
-                      <IconButton
-                        type="button"
-                        onClick={() => remove(i)}
-                        sx={{ mx: 1, my: 0, color: 'red', borderRadius: '4px', outline: 'solid' }}
-                      >
-                        <DeleteIcon />
-                      </IconButton>
-                      <Box></Box>
-                    </Stack>
-                  </Box>
-                </Grid>
-              </>
-            );
-          })}
-        </Grid>
-      </Stack>
-    </>
+    <Box>
+      <Grid container spacing={2}>
+        {ls.map((_element, i) => {
+          return (
+            <Grid item sx={{ alignItems: 'center' }} xs={12} md={4}>
+              <InputLabel>{bulletName + ' ' + (i + 1)}</InputLabel>
+              <Box sx={{ display: 'flex', alignItems: 'start' }}>
+                <TextField
+                  required
+                  multiline
+                  placeholder={'Enter a ' + bulletName + '...'}
+                  inputProps={{ style: { fontSize: 14 } }}
+                  minRows={2}
+                  autoComplete="off"
+                  sx={{ width: 12 / 12, padding: 0 }}
+                  {...register(`${name}.${i}.detail`)}
+                />
+                <IconButton
+                  type="button"
+                  onClick={() => remove(i)}
+                  sx={{ mx: 1, color: 'red', marginLeft: '15px', borderRadius: '4px', outline: 'solid' }}
```

Comment:
can you add a marginTop of '3px' to this so it lines up better with the text input